js/ui/tooltips/number-scrubber.js

```diff
@@ -79,6 +79,14 @@ TooltipEngine.classes.numberScrubber = TooltipBase.extend({
                     self.$el.addClass("dragging");
                     $(this).css("visibility", "hidden");
                     self.trigger("scrubbingStarted");
+                    // The text-to-be-tweaked needs to be the same length at the start and end
+                    // of the anti-undo changes.
+                    // I could probably just remember the length, but I like putting back the
+                    // original string. (It might even matter for i18n.)
+                    var Range = ace.require("ace/range").Range;
+                    var loc = self.aceLocation;
+                    var range = new Range(loc.row, loc.start, loc.row, loc.start + loc.length);
+                    self.originalString = parent.editor.getTextRange(range);
```

Comment:
I need to fix that suggestion message.  I forgot that not everything in external/ comes from submodules.  That particular dependency comes from bower.  If you don't have bower installed you can install it by running: ```npm install -g bower```.  Then run ```bower install``` to install the dependencies.